src/App.jsx

```diff
@@ -16,66 +16,68 @@ export const goods = [
 ];
 
 export const App = () => {
-  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [value, setValue] = useState('Jam');
 
   const clearSelection = () => {
-    setSelectedGood('');
+    setValue('');
   };
 
   return (
     <main className="section container">
-      <h1 className="title is-flex is-align-items-center">
-        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
-        {selectedGood && (
+      {value.length > 0 ? (
+        <h1 className="title is-flex is-align-items-center">
+          {value} is selected
           <button
             data-cy="ClearButton"
             type="button"
             className="delete ml-3"
             onClick={clearSelection}
           />
-        )}
-      </h1>
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
 
-      {goods.map(item => (
-        <table className="table" key={item}>
-          <tbody>
+      <table className="table">
+        <tbody>
+          {goods.map(item => (
             <tr
+              key={item}
               data-cy="Good"
-              className={
-                selectedGood === item ? 'has-background-success-light' : ''
-              }
+              className={value === item ? 'has-background-success-light' : ''}
```

Comment:
'value === item'
you can create a variable to avoid repetition

Reply:
Good point, fix this
But also use [classnames](https://www.npmjs.com/package/classnames) here